radix-engine/src/kernel/id_allocator.rs

```diff
@@ -38,6 +39,7 @@ impl IdAllocator {
         Ok(())
     }
 
+    #[trace_resources]
```

Comment:
Suggest remove this tracing. It's just part of the `kernel_allocate_node_id` implementation.